### ISSUE TITLE
Scope packages aren't handled correctly

### ISSUE THREAD

**Issue by stefanbuck** [danielbayley/atom-hyperclick-octolinker#3]
Given I have the following `package.json` file, the plugin doesn't include the scope `@google-cloud` when hover over a dependency. This cause an internal server error for the first package `debug-agent` and a wrong redirect for the second package.

```json
{
  "dependencies": {
    "@google-cloud/debug-agent": "^2.1.3",
    "@google-cloud/storage": "^1.2.1"
  }
}
```

**Comment by stefanbuck** [comment]
That was quick, thanks

**Comment by danielbayley** [comment]
Released in 0.1.3.
